apps/files/src/components/AllFilesList.vue

```diff
@@ -118,9 +118,17 @@ export default {
 
     $_emptyFolderText () {
       if (this.$_isFavoritesList) {
-        return this.$gettext('No files or folders marked as favorite')
+        return this.$gettext('There are no resources marked as favorite.')
       } else {
-        return this.$gettext('Folder is empty')
+        return this.$gettext('There are no resources in this folder.')
+      }
+    },
+
+    $_emptyFolderSubText () {
+      if (this.$_isFavoritesList) {
+        return this.$gettext('You can mark some by clicking on the star icon.')
+      } else {
+        return this.$gettext('Drag files and folder here to upload, or browse files from your computer.') // TODO: link
```

Comment:
- `folder` missing an s (plural)
- maybe replace `computer` with `device`, as we're also targeting mobile